corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -503,7 +503,7 @@ class Assertion(XmlObject):
 
 class Validation(XmlObject):
     ROOT_NAME = 'validation'
-    xpath = XPathField('@xpath')
+    test = XPathField('@test')
```

Comment:
Two attributes, which differ by one letter, seems error-prone, but I defer to you & Shubham on final XML spec. (Possible alternative: `function` instead of `test`?)

Reply:
No strong feelings here - I'd originally gone with `xpath`, but `Assertion` (just above this) also does `test` and `text`, so this seemed reasonable too.  @shubham1g5 any thoughts?

Reply:
ah, I didn't remember `Assertion` already does this, that lessens my complaint

Reply:
Yeah we used `test` to make them consistent with assertions.